src/Chapter13/Listing13.25.ExamingingAnExpressionTree.cs

```diff
@@ -56,5 +57,6 @@ expression.NodeType switch
                 _ => expression.ToString() +
                     " (" + expression.NodeType.ToString() + ")",
             };
+        #endregion INCLUDE
```

Comment:
Is this correct? This will leave unbalanced curly braces since the `#region INCLUDE` includes the opening brace, but this one comes before the closing brace on line 61